app/src/util/tools/formatting.ts

```diff
@@ -1,20 +1,35 @@
 import { BigNumber, formatUnits } from 'ethers/utils'
 
 export const formatBigNumber = (value: BigNumber, decimals: number, precision = 2): string => {
+  // console.log('current formatting', Number(formatUnits(value, decimals)).toFixed(precision))
+
   return Number(formatUnits(value, decimals)).toFixed(precision)
 }
+export const bigNumberToString = (value: BigNumber | string, decimals: number, precision = 2): string => {
+  const bigNumner = bigNumberToNumber(value, decimals)
```

Comment:
Once we get the number value here we can just wrap it in `formatNumber`

Reply:
@KadenZipfel Hmm well the issue is that formatNumber accepts string as its number argument...
![Screenshot 2021-07-09 at 15 50 23](https://user-images.githubusercontent.com/33226956/125088092-60b00d00-e0cd-11eb-88aa-ecaca4e7f94d.png)
if we are going to go that route maybe something like this then makes more sense?
![Screenshot 2021-07-09 at 15 53 52](https://user-images.githubusercontent.com/33226956/125088599-dd42eb80-e0cd-11eb-8bdd-4e790c43a347.png)

Reply:
Yessir that looks ideal!